isis/appdata/translations/voyager.trn

```diff
@@ -0,0 +1,204 @@
+Group = MissionPhaseName
```

Comment:
Why can't this go in `isis/voyager/apps/voy2isis` ?

Reply:
They can I put them in the wrong place.  Now moved